### PR TITLE
docs: add IDE specific configs for tailwindcss

### PR DESCRIPTION
**Pull request by a-mesin**
adds configs from #230 into docs

### CODE REVIEW

File: docs/docs/09-commands-and-tools/02-ide-support.md [hunk 1]
```diff
@@ -36,20 +62,38 @@ end
...
+        templ = "html"
+    }
+  }
 })
 ```
 
 ## Troubleshooting
 
-### Check that go, gopls and templ are installed and are present in the path.
+### Check that go, gopls and templ are installed and are present in the path
```
Comment (a-h) [inline docs/docs/09-commands-and-tools/02-ide-support.md:96]:
```suggestion
### Check that go, gopls and templ are installed and are present in the path.
```

**Comment by a-h** [comment]
Thanks very much!

**Comment by pistolpeter** [comment]
Hi, the config for Tailwind and Neovim was entered incorrectly. Lines 86 and 87 need to be swapped.

**Comment by a-mesin** [comment]
@pistolpeter thanks for pointing out, created another PR to fix it

**Pull request merged**